xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/RegistrationConfig.xml

```diff
@@ -522,9 +522,30 @@
       <passwordRuleOneUpperCaseEnabled>0</passwordRuleOneUpperCaseEnabled>
     </property>
     <property>
-      <registrationSuccessMessage>#set($message = $services.localization.render('core.register.successful', 'xwiki/2.1', ['USERLINK', $userName]))
+      <registrationSuccessMessage>#set($discard = $xwiki.ssx.use("XWiki.RegistrationConfig"))
+#set($displayName = "$!firstName $!lastName")
+#set($noReadableName = ($!firstName == "") || ($!lastName == ""))
+#if($noReadableName)
+  #set($displayName = $userName)
+#end
+#set($headline = $services.localization.render('core.register.successful.welcome', [$displayName]))
 #set($userLink = $xwiki.getUserName("$userSpace$userName"))
-{{info}}$message.replace('USERLINK', "{{html clean=false}}$userLink{{/html}}"){{/info}}</registrationSuccessMessage>
+#set($successAndLogin = $services.localization.render('core.register.successful.successandlogin'))
+[[image:undraw_done_re_oak4.svg||data-xwiki-image-style-alignment="center" height="50vh"]]
+
+{{html clean="true"}}
```

Comment:
clean="true" is the default AFAIK

Reply:
:+1: https://github.com/xwiki/xwiki-platform/pull/3155/commits/abb32a7e17aa0446a5fd4ff277623673c8747624